packages/playground/wordpress/src/boot.ts

```diff
@@ -201,7 +203,11 @@ export async function bootWordPress(
 		);
 	}
 
-	if (options.wordPressZip && !options.dataSqlPath) {
+	const shouldInstall =
+		(options.installWordPress ?? Boolean(options.wordPressZip)) &&
+		!options.dataSqlPath;
```

Comment:
I don't like the naming and the semantics here. If we override, then having `dataSqlPath` should not affect it further. Perhaps this would work better?

```suggestion
	const shouldInstall =
		options.installWordPress ??
		(Boolean(options.wordPressZip) && !options.dataSqlPath);
```

Reply:
Ah, no, I see. `dataSqlPath` is an "SQL file to load **instead** of installing WordPress". Dang. Okay, the initial thing was okay. I supposed I'd rather introduce an explicit `installationMode` option with three possible states: `"wp-installer" | "sql-dump" | false`.